acitoolkit/acibaseobject.py

```diff
@@ -964,6 +964,30 @@ def _get_url_extension(self):
 
     def __str__(self):
         return self.name
+    
```

Comment:
Does this have to be in the base class ? Can it be moved to Tenant class ? Also, should it just be named get_from_json() ?